cscs-checks/microbenchmarks/mpi/mpi_helloworld.py

```diff
@@ -0,0 +1,36 @@
+import os
+import reframe.utility.sanity as sn
+
+from reframe.core.deferrable import deferrable
+from reframe.core.pipeline import RegressionTest
+
+
+class MPIHelloWorldTest(RegressionTest):
+    def __init__(self, **kwargs):
+        super().__init__('mpi_helloworld_check',
+                         os.path.dirname(__file__), **kwargs)
+        self.valid_systems = ['daint:gpu', 'daint:mc',
+                              'dom:gpu', 'dom:mc',
+                              'kech:cn', 'kech:pn',
+                              'leone:normal', 'monch:compute']
```

Comment:
Have you tested it in all those systems? CI won't do that for you, cos it's not a production test.

Reply:
I didn't realize that. `kesch` was misspelled.